kuma/javascript/src/search-results-page.jsx

```diff
@@ -171,13 +184,27 @@ export class SearchRoute extends Route<SearchRouteParams, SearchResults> {
                         );
                     }
                 })
-                .then(
-                    results =>
-                        results && {
-                            results: results.hits.hits.map(h => h._source),
-                            error: null
-                        }
-                )
+                .then(results => {
+                    if (
+                        !results ||
+                        !results.hits ||
+                        !Array.isArray(results.hits.hits)
+                    ) {
+                        throw new Error('Search API returned unexpected data');
+                    }
+                    return {
+                        results: results.hits.hits.map(hit => {
+                            let score = hit._score;
+                            let excerpt =
+                                hit.highlight &&
+                                hit.highlight.content &&
+                                hit.highlight.content[0];
```

Comment:
First of all, we do ask ES to highlight on `['summary', 'content']` but we're never displaying any highlights from `summary`. 

Somewhere in here I think we should do something like:
```
const lines = []
['summary', 'highlight'].forEach(ns => {
  (hit.highligth[ns] || []).forEach(match => {
    if (lines.length < 3) {
      lines.push(<p key={ns + match} dangerouslySetInnerHTML={{ __html: match}}/>);
    }
  });  
});
```
It's not right but the point is that we display highlights from summary (first) and fill in from content up to a max point.